src/rebuild/initiator.c

```diff
@@ -223,7 +223,7 @@ rebuild_fetch_update_bulk(struct rebuild_one *rdone, daos_handle_t oh,
 	}
 
 end:
-	vos_update_end(ioh, rdone->ro_version, &rdone->ro_dkey, rc, NULL);
+	rc = vos_update_end(ioh, rdone->ro_version, &rdone->ro_dkey, rc, NULL);
```

Comment:
will it over-write the rc in above code? may use ret and rc = rc ? : ret;?

Reply:
The vos_update_end() will keep and return the original err# if it is non-zero.